README.md

```diff
@@ -1,15 +1,226 @@
+      - [Component tools](#component-tools)
+  - [Additional notes](#additional-notes)
+  - [Help/FAQ/Troubleshooting](#helpfaqtroubleshooting)
+  - [Acknowledgements/citations/credits](#acknowledgementscitationscredits)
+
+## Description
+Somatic-shortV-nf is a pipeline for identifying somatic short variant (SNPs and indels) events in human Illumina short read whole genome sequence data from tumour and matched normal BAM files. The pipeline follows the [GATK's Best Practices](https://gatk.broadinstitute.org/hc/en-us/articles/360035894731-Somatic-short-variant-discovery-SNVs-Indels-) workflow. 
+The pipeline is written in Nextflow and uses Singularity/Docker to run containerised tools.
+
+There are two main steps to this workflow 
+- First step is to generate a large set of candidate somatic variants using the tool  [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132). 
+- The next step is to filter the candidate variants to obtain a more confident set of somatic variant calls. 
+- The third (optional) step is to annotate the variants with gene level information using an external genomic variant annotations and functional effect prediction tool called [SnpEff](http://pcingola.github.io/SnpEff/).
+
+## Diagram
+
+<p align="center"> 
+<img src="./images/Somatic_variant_calling.png" width="60%">
+</p> 
+
 
 ## Workflow description 
 
 ## User guide 
 
-## Component tools 
+To run this pipeline, you will need to prepare your input files, reference data, and clone this repository. Before proceeding, ensure Nextflow is installed on the system you're working on. To install Nextflow, see these [instructions](https://www.nextflow.io/docs/latest/getstarted.html#installation). 
+
+### 1. Prepare inputs
+
+To run this pipeline you will need the following inputs: 
+
+* Paired Tumor-Normal paired BAM files
+* Corresponding BAM index files  
+* Input sample sheet 
+
+This pipeline processes paired BAM files and is capable of processing multiple samples in parallel. BAM files are expected to be coordinate sorted and indexed (see [Fastq-to-BAM](https://github.com/Sydney-Informatics-Hub/Fastq-to-BAM) for an example of a best practice workflow that can generate these files).  
+
+You will need to create a sample sheet with information about the samples you are processing, before running the pipeline. This file must be **comma-separated** and contain a header and one row per sample. Columns should correspond to sampleID, BAM-N file-path, BAM-T file-path: 
+
+```csv
+sampleID,bam-N,bam-T 
+SAMPLE1,/data/Bams/sample1-N.bam,/data/Bams/sample1-T.bam
+SAMPLE2,/data/Bams/sample2-N.bam,/data/Bams/sample2-T.bam|
+``````
+
+When you run the pipeline, you will use the mandatory `--input` parameter to specify the location and name of the input file: 
+
+```
+--input /path/to/samples.csv
+```
+
+### 2. Prepare the reference materials 
+
+To run this pipeline you will need the following reference files:
+
+* Indexed reference genome in FASTA format 
```

Comment:
Clarify what indexes are required (samtools, bwa, gatk?)

Reply:
Please refer to https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf/pull/2#discussion_r1432015491 below